scrapper.py

```diff
@@ -35,7 +45,23 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 13_2_3 like Mac OS X) '
+            'AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.3 Mobile/15E148 Safari/604.1',
+            'Accept-Encoding': 'gzip, deflate, br',
+            'Accept-Language': 'ru,en;q=0.9,de;q=0.8'
+        }
+
+        for seed_url in self.seed_urls:
+            response = requests.get(url=seed_url, headers=headers)
+            time.sleep(3)
+            if not response.ok:
+                print("Something went wrong...")
```

Comment:
it is a weak check. probably we do not want to process such an URL then? what do you think?